src/main/kotlin/no/nav/familie/ef/sak/repository/domain/VilkårVurdering.kt

```diff
@@ -0,0 +1,39 @@
+package no.nav.familie.ef.sak.repository.domain
+
+import org.springframework.data.annotation.Id
+import org.springframework.data.relational.core.mapping.Column
+import org.springframework.data.relational.core.mapping.Embedded
+import org.springframework.data.relational.core.mapping.Table
+import java.util.*
+
+@Table("vilkar_vurdering")
+data class VilkårVurdering(
+        @Id
+        val id: UUID = UUID.randomUUID(),
+        @Column("behandling_id")
+        val behandlingId: UUID,
+        val resultat: VilkårResultat = VilkårResultat.IKKE_VURDERT,
+        val type: VilkårType,
+        val begrunnelse: String? = null,
+        val unntak: String? = null,
+        @Embedded(onEmpty = Embedded.OnEmpty.USE_EMPTY)
+        val sporbar: Sporbar = Sporbar()
```

Comment:
Må vi bruke `?` etter sporbar hvis den kan være "Empty" i databasen? 
Hvis ikke risikrerer vi kanskje å få en null-verdi - og Kotlin blir neppe så glad for det?

Eller har jeg misforstått hvordan `Embedded.OnEmpty.USE_EMPTY` fungerer?

Reply:
Vet ikke, vi må ha en onEmpty, USE_EMPTY/USE_NULL finnes, og kanskje vi burde sette USE_NULL og att den smeller hvis den ikke finnes. Den skall alltid finnes

Reply:
USE_NULL finnes, ja. Da er jeg brått usikker på hva USE_EMPTY gjør? 🤔